integration_tests/steps/pv_tests.py

```diff
@@ -17,9 +17,8 @@
 """
 from behave import then
 from math import isclose
-from gsy_framework.constants_limits import ConstSettings
+from gsy_framework.constants_limits import ConstSettings, DEFAULT_PRECISION
```

Comment:
What would you think about removing `DEFAULT_PRECISION` and directly using `limit_float_precision`?

Reply:
yes, will do it.